packages/account-v2/package.json

```diff
@@ -14,7 +14,7 @@
     },
     "dependencies": {
         "@deriv-com/ui": "^1.14.5",
-        "@deriv-com/utils": "^0.0.20",
+        "@deriv-com/utils": "^0.0.23",
```

Comment:
Versioning below number 1 is using used for `alpha` version of programs. I do not think if utils is an `alpha` version.